src/components/Todo/Todo.jsx

```diff
@@ -0,0 +1,50 @@
+// components
+import { useState } from 'react';
+import { TodoForm } from '../TodoForm/TodoForm';
+import { TodoList } from '../TodoList/TodoList';
+
+// style
+import style from './Todo.module.scss';
+
+export function Todo() {
+  const [todos, setTodos] = useState([]);
+
+  const addTodo = (todo) => {
+    setTodos([todo, ...todos]);
+  };
+
+  const toggleTodo = (id) => {
+    setTodos(
+      todos.map((todo) => {
+        if (todo.id === id) {
+          return {
+            ...todo,
+            completed: !todo.completed,
+          };
+        }
+
+        return todo;
+      })
+    );
+  };
+
+  const deleteTodo = (id) => {
+    setTodos(todos.filter((todo) => todo.id !== id));
+  };
+
+  return (
+    <>
+      <div className={style.todo}>
+        <div className={style['todo-body']}>
```

Comment:
I think for consistency you should use camelCase, like todoBody, and style.todoBody - same approach like string above